config/config.go

```diff
@@ -79,8 +79,7 @@ func New(cfg *config.C) (*Config, error) {
 
 func defaultConfig() (*Config, error) {
 	ret := &Config{
-		Period:    4 * time.Hour,
-		Benchmark: CIS_K8S,
```

Comment:
why is it needed? all integrations besides vulnerability management are sending benchmark type.
It causes us to enrich vm config with the wrong benchmark (`cis_k8s`).

Reply:
Does it cause a change in the schema of the events from other integrations as well?

Reply:
All other integrations send the `benchmark` value so they are not affected by the default config.